pkg/pquerier/query_integration_test.go

```diff
@@ -3286,6 +3286,460 @@ func (suite *testQuerySuite) TestSelectDataframeDaownsampleMetricsHaveBigGaps()
 	suite.Require().Equal(1, dataFrameCount, "series count didn't match expected")
 }
 
+func (suite *testQuerySuite) TestClientWindowedAggregationWindowBiggerThanStep() {
```

Comment:
This file contains TOO many tests. Consider to break it into categories.

Reply:
take a look at https://github.com/v3io/v3io-tsdb/pull/326 😄

Reply:
break it into smaller logical units/files